core_backend/tests/rails/test_language_indentification.py

```diff
@@ -23,14 +24,16 @@ def available_languages() -> list[str]:
 def read_test_data(file: str) -> List[Tuple[str, str]]:
     """Reads test data from file and returns a list of strings"""
 
-    with open(file, "r") as f:
+    file_path = Path(__file__).parent / file
```

Comment:
@suzinyou - I updated how the data files are loaded so hopefully this should work on your setup without any fiddling